### DISCUSSION TITLE
Using the TFT display and the ST7735s driver

### DISCUSSION THREAD

**Discussion by coderatchance** [SpenceKonde/megaTinyCore#969] [Q&A]
Hello all,

I am trying to get around the TFT mini display based on the ST7735s driver with the AT Tiny 3216. 
Tried the basic [adafruit gfx library](https://github.com/adafruit/Adafruit-GFX-Library) and it works without a problem... however, apparently this library is not very [efficient](https://forum.arduino.cc/t/faster-libraries-for-st7735/1082722/3), so, moved on to the [PDQ_GFX](https://hackaday.io/project/6038-pdqgfx-optimzed-avr-lcd-graphics/details), but having a problem with the compilation, namely installation of the library that can be found [here](https://github.com/XarkLabs/PDQ_GFX_Libs)  . 

Any ideas how to get around this?

I guess the PDQ_GFX will work on the megaTinyCore!

Best.

**Comment by SpenceKonde** [comment]
lease be more specific about the problem you're having; I would expect to be able to download either a "released" version or the current github version as a .zip or .tar.gz, extract that with favorite archiving tool, and copy it to the libraries folder inside your sketchbook folder. Do note the warning at the bottom of that lib

> Currently, the library may only be used from the INO file in your project. You cannot include it in a header file and use it from other CPP files. The current workaround is to write wrapper functions or classes, declare them in a header file, and then implement them in the INO file.

So what is it that makes you believe that you have not installed that library correctly? Or that there is any problem whatsoever? You didn't mention any problems with the image on the screen so what is the problem? Does it not get there and fail to compile? What is the error message? Just because you don't understand the error doesn't mean I won't. I know both the tools and parts a lot better than most folks around here. Butall I see when I look into my crystal ball isa distorted view of the space behind it - I cant see thebehavior you are seeing, making it impossible to characterize and startspeculating about the cause of.
